utils/logger.py

```diff
@@ -1,4 +1,4 @@
-import logging, sys, os
+import logging, logging.handlers, sys, os
```

Comment:
FYI. PR#36 also updates this line in addition to line 97 to fix a compatibility bug with recent versions of python and the way to invoke the file handler.